src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -397,6 +427,52 @@ bool Profile::IsDynamicProfileObject(const Json::Value& json)
     return !source.isNull();
 }
 
+// Function Description:
+// - Apply each of the current Profile's settings values onto the given Profile, if it is set
+// Arguments:
+// - profile: the Profile to be updated
+// Return Value:
+// - <None>
+void Profile::ApplyTo(Profile* profile) const

```

Comment:
reminder to scrub applyto! :smile:

Reply:
Idea: using `ApplyTo()` to make `Copy()` easier. Less repeated code!